src/components/apphost.js

```diff
@@ -1,4 +1,4 @@
-
+import { version as appVersion } from '../../package.json';
```

Comment:
```suggestion
import { version as appVersion } from '~/package.json';
```
Not tested.

Reply:
That doesn't work :slightly_smiling_face: 

```
ERROR in ./components/apphost.js 9:0-56
Module not found: Error: Can't resolve '~/package.json' in '/home/bill/Projects/jellyfin-web/src/components'
 @ ./scripts/site.js 33:0-48 144:9-25 207:6-22 212:7-23
```